download_models/downloadOVMSModels.sh

```diff
@@ -1,18 +1,12 @@
 #!/bin/bash
 #
-# Copyright (C) 2023 Intel Corporation.
+# Copyright (C) 2024 Intel Corporation.
```

Comment:
this may be a change that will have to be done in other files as well. Out of scope for this PR though.

Reply:
I just changed whenever I touched the files